packages/mui-joy/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.27.0",
-    "@mui/base": "workspace:*",
+    "@mui/base": "7.0.0-beta.4",
```

Comment:
Uff, so the version of @mui/base-ui that lives on the v6 branch will actually be 7.0.0-beta.4? Confusing 😄

Reply:
I am honeslty not sure where this version came from. Maybe @DiegoAndai or @aarongarciah can confirm it, but I hopnestly don't think this version matters here, because we are not releasing Joy UI from here, so it's kind of a "dead code" anyway.

Reply:
The reason for `7.0.0-beta.4` is explained in https://github.com/mui/material-ui/pull/45565#discussion_r1993445810

> We need a `@mui/base` release with the new package layout because `@mui/lab` depends on it. Otherwise, `@mui/lab` will fail to import in Node (ESM module with a fake-ESM dependency).

Reply:
On, now I remember why I bumped to this version, it was related to esm, bumping it to 5.0.0-beta.40-1 fails with: https://app.circleci.com/pipelines/github/mui/material-ui/152005/workflows/3b5d02dd-a8d2-4584-a16f-4ec612aa988d/jobs/816592. The latest released by depend on 5.0.0-beta.40-1, but it was not released from this branch. I am reverting back to 7.0.0-beta.4 (it's the same reasoning as @mui/lab).